single_tech_samples/synapseanalytics/sample3_integration_testing/tests/integrationtests/README.md

```diff
@@ -0,0 +1,71 @@
+# Integration test suite for ADF
```

Comment:
a section troubleshooting would be helpful for common issues.

I ran into one where ODBC failed to connet... i fixed the .env variable values, and then had to rebuild the container. this may not be very obvious to everyone though.

Reply:
I think after we screen shared, we learned that one of the environment variables needed to be something else. I updated the infrastructure so that the dev.env file contains all required environment variables for the integration test so this should be a lot easier now (just copy and paste).

It shouldn't be the case that a rebuild is necessary for the integration test dev container since it uses Python's os package which looks for a `.env` file on every run. I'm happy to add a Troubleshooting section, just not too sure what all to put in it.